content/reference-architectures/cold-start.md

```diff
@@ -0,0 +1,627 @@
+```shell
+export AWS_VAULT_BACKEND=file
+aws-vault add cpco
+```
+{{% /dialog %}}
+
+{{% dialog type="info" icon="fa-info-circle" title="Note" %}}
+Replace the profile name `cpco` with your own (for consistency, we recommend using the same name as the namespace in the Terraform modules).
+
+You should set `AWS_VAULT_BACKEND=file` in your shell `rc` config (e.g. `~/.bashrc`) so it persists.
+{{% /dialog %}}
+
+* Keep in mind that AWS has limits on the number of accounts in an organization (see [Organization reference limits](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_reference_limits.html)).
+If you are planning on provisioning all the accounts (`prod`, `staging`, `dev`, `audit`, `testing`), contact AWS Support and request a limit increase.
+
+
+# Cold Start Process
+
+In this example, we'll provision resources for the following two accounts:
+
+* `root` - always in use as the Root of the AWS accounts hierarchy and also as the main account to store all the IAM users and roles to access the other accounts
+* `testing` - we use it to quickly create and destroy infrastructure for testing 
+
+The other stages (`prod`, `staging`, `dev`, `audit`) are similar (they might differ by the resources you provision), and as has been noticed before, you might not need all of them.
+
+
+## Copy Cloud Posse Reference Architectures
+
+Copy the [root](https://github.com/cloudposse/root.cloudposse.co) and [testing](https://github.com/cloudposse/testing.cloudposse.co) repos to your local workstation
+into two different folders. For this example, we'll use `~/root.cloudposse.co` and `~/testing.cloudposse.co` respectively.
+
+Update all ENV variables in the two `Dockefiles` in the repos with the values for your project:
+ 
+ * Change `DOCKER_IMAGE`
+ * Replace the namespace `cpco` with your own in all ENV vars
+ * Change the domain names from `cloudposse.co` to your own
+ * In root, update the account ID (`TF_VAR_account_id`) to your own `root` account ID
+ * Change the IAM user names for the accounts
+ * Update the account emails
+ * In `testing`, select only the resources you need to provision (using `COPY --from=terraform-root-modules`)
+
+
+## Add AWS Profile For `root`
+
+We require all users to assume IAM roles to access all the accounts (including `root`).
+
+But since we start with a new account with no users and roles created yet, in this step we use the `admin` user credentials to access the account without assuming roles.
+
+We'll update this profile to use `role_arn` later after we provision the roles.
+
+In `~/.aws/config` file, add this profile for the `root` account:
+
+{{% dialog type="code-block" icon="fa fa-code" title="Add AWS Profile For `root`" %}}
+```
+[profile cpco-root-admin]
+region=us-west-2
+mfa_serial=arn:aws:iam::323330167063:mfa/admin@cloudposse.co
+source_profile=cpco
+
+# This profile is required by aws-vault where it stores the Access Key ID and Secret Access Key as explained in Prerequisites
```

Comment:
Add chapter link

Reply:
this is a comment, chapter link will not work.